src/PrestaShopBundle/Resources/config/routing/admin/improve/international/country.yml

```diff
@@ -17,6 +17,14 @@ admin_countries_search:
     _legacy_controller: AdminCountries
 #    _legacy_link: AdminCountries:submitFiltercountry
 
+admin_countries_create:
+  path: /new
+  methods: [ GET, POST ]
+  defaults:
+    _controller: 'PrestaShopBundle\Controller\Admin\Improve\International\CountryController::createAction'
+    _legacy_controller: AdminCountries
+#    _legacy_link: AdminZones:addCountry
```

Comment:
```suggestion
#    _legacy_link: AdminCountries:addCountry
```

Reply:
Indeed I didn't see this one but we must set it correctly from the start it will avoid future mistakes when the page is enabled.